feder/static/css/style.css

```diff
@@ -7443,6 +7443,27 @@ button.close {
     background-color: #D72626; }
   .gray .page-title {
     color: #D72626; }
+  .gray .metadata-button-container {
+    text-align: right; }
+    .gray .metadata-button-container > .navbar-toggle {
+      padding-top: 2px;
+      padding-bottom: 2px;
+      color: #fff;
+      background-color: #bb2121;
+      border-color: #a51d1d;
+      float: none;
+      display: inline-block; }
+      .gray .metadata-button-container > .navbar-toggle .icon-bar :hover, .gray .metadata-button-container > .navbar-toggle .icon-bar :focus {
+        background-color: #fff; }
+      .gray .metadata-button-container > .navbar-toggle .icon-bar + .icon-bar {
+        margin-top: 1px; }
+      .gray .metadata-button-container > .navbar-toggle span {
+        line-height: 75%; }
+      .gray .metadata-button-container > .navbar-toggle .icon-bar {
+        background-color: #fff; }
+        .gray .metadata-button-container > .navbar-toggle .icon-bar.first {
+          padding-top: 0px;
```

Comment:
@mik-laj , czy to jest ok?

Reply:
Moim zdaniem element navbar-toggle powinien być tylko w navbarze.

Reply:
@mik-laj wybrałem mniejsze zło - style navbar-toggle trzeba byłoby powtórzyć, a trochę tego jest (focus, hover). Uwazasz, że warto?

Reply:
Moim zdaniem warto powtórzyć, bo tworzenie takich klas to prosta droga do spaghetti. Za pół roku patrzysz w kod HTML i już nie wiesz dlaczego tam jest `.navbar-toggle` i nie potrafisz tego też zmodyfikować. Nie patrzyłem jak to wygląda, ale też podejrzewam, że to nie pasuje idealnie do tej sytuacji.

Reply:
OK, poprawione